backend/app/Http/Controllers/AuthController.php

```diff
@@ -58,7 +58,17 @@ public function login(Request $request)
             'user' => $user,
             'token' => $token
         ];
-
         return response($response, 201);
     }
+
+    public function logout(Request $request)
+    {
+        $id = $request->validate([
+            'user_id' => ['required', 'numeric']
+        ]);
+        $user = User::findOrFail($id)->first();
+        $user->tokens()->delete();
+
+        return response()->json(['success' => 'Logged Out'], 201);
+    }
```

Comment:
You don't need to pass the Id to be logout because there is a `Auth` facade  and  `auth()` helper, which can retrieve the authenticated user. 
Here is the related topic to this one on laravel docs
https://laravel.com/docs/7.x/authentication#retrieving-the-authenticated-user